rtbkit/common/plugin_table.h

```diff
@@ -0,0 +1,109 @@
+/* plugin_table.h
+   Flavio Moreira, 15 December 2014
+   Copyright (c) 2014 Datacratic.  All rights reserved.
+*/
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <boost/any.hpp>
+#include "jml/arch/spinlock.h"
+
+namespace RTBKIT {
+
+struct PluginTable
```

Comment:
can I merge this?


On Wed, Jan 7, 2015 at 10:50 PM, Eric Robert <notifications@github.com>
wrote:

> In rtbkit/common/plugin_table.h
> <https://github.com/datacratic/rtbkit/pull/116#discussion-diff-22634427>:
>
> > @@ -0,0 +1,109 @@
> > +/* plugin_table.h
> > +   Flavio Moreira, 15 December 2014
> > +   Copyright (c) 2014 Datacratic.  All rights reserved.
> > +*/
> > +
> > +#pragma once
> > +
> > +#include <string>
> > +#include <unordered_map>
> > +#include <boost/any.hpp>
> > +#include "jml/arch/spinlock.h"
> > +
> > +namespace RTBKIT {
> > +
> > +struct PluginTable
>
> Looks good to me :)
> … <#14ac7aa8568b9e67_>
> On Wed, Jan 7, 2015 at 11:21 AM, oktal <notifications@github.com> wrote:
> In rtbkit/common/plugin_table.h <
> https://github.com/datacratic/rtbkit/pull/116#discussion-diff-22596041>:
> > @@ -0,0 +1,109 @@ > +/* plugin_table.h > + Flavio Moreira, 15 December
> 2014 > + Copyright (c) 2014 Datacratic. All rights reserved. > +*/ > + >
> +#pragma once > + > +#include <string> > +#include <unordered_map> >
> +#include <boost/any.hpp> > +#include "jml/arch/spinlock.h" > + >
> +namespace RTBKIT { > + > +struct PluginTable Ok so now you have one static
> instance of the PluginTable per Factory type ? I think I'm ok with that,
> @EricRobert <https://github.com/EricRobert> <https://github.com/EricRobert>
> ? — Reply to this email directly or view it on GitHub <
> https://github.com/datacratic/rtbkit/pull/116/files#r22596041>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datacratic/rtbkit/pull/116/files#r22634427>.
>